tidyblocks/gui.js

```diff
@@ -105,6 +105,11 @@ class GuiEnvironment {
     document.getElementById('statsOutput').innerHTML = stats2table(values, legend)
   }
 
+  displayStatsPlot (spec1, spec2) {
+    vegaEmbed('#statsPlot1', spec1, {})
```

Comment:
Why two plots?

Reply:
We're making the same graph twice except that:

- In plot 1 we use `result.statistic` 
- In plot 2 we use (`result.xmean - result.ymean`). 

Maybe I should make the spec into a ttestPlot function that we can then run on the array of the two result values, then vegaEmbed can plot the two specs?

Reply:
so 'base' and 'overlay' or...? I'll never remember what '1' and '2' are - but I'm happy to merge this now if it's working.